script.js

```diff
@@ -28,4 +29,4 @@ function createCalendar(ID, year, month) {
   document.getElementById(ID).innerHTML = html
```

Comment:
It is hard to keep in mind what is in temp variable. Also, it is too general name. Use more variables instead.